modules/ROOT/pages/writing.adoc

```diff
@@ -711,15 +714,18 @@ Given the following example:
       .option("url", SparkConnectorScalaSuiteIT.server.getBoltUrl)
       .option("labels", ":Person:Customer")
       .option("node.keys", "surname")
-      .option("schema.optimization.node.unique", true)
+      .option("schema.optimization.node", "UNIQUE")
```

Comment:
```suggestion
      .option("schema.optimization.node.key", "UNIQUE")
```

Reply:
done

Reply:
my bad, it's `.keys` and I don't see the change